lib/sprockets/less/v4/importer.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+require_relative '../v3/importer'
+module Sprockets
+  module Less
+    module V4
+      # class used for importing files from SCCS and SASS files
+      class Importer < Sprockets::Less::V3::Importer
+        def syntax_mime_type
+          "text/#{syntax}"
+        end
+
+        def engine_from_glob(glob, base_path, options)
+          context = options[:custom][:sprockets_context]
+          engine_imports = resolve_glob(context, glob, base_path).reduce(''.dup) do |imports, path|
+            context.depend_on path[:file_url]
+            relative_path = path[:path].relative_path_from Pathname.new(base_path).dirname
+            imports << %(@import "#{relative_path}";\n)
+          end
+          return nil if engine_imports.empty?
+          ::Sass::Engine.new engine_imports, options.merge(
```

Comment:
THe v4 importer is not ready yet, i copied it from sprockets-sass aparently, I need to replace Sass::Engine with Less::Parser,   I still need to refactor this class completely, I almost fixed the v4 importer for SASS...i have 2 tests that don't want to run there, but i will try to make this work too.